pyregion/core.py

```diff
@@ -37,16 +45,31 @@ def __getslice__(self, i, j):
         return self[max(0, i):max(0, j):]
 
     def check_imagecoord(self):
+        """Are all shapes in image coordinates?
+
+        Returns ``True`` if yes, and ``False`` if not.
+        """
         if [s for s in self if s.coord_format != "image"]:
             return False
         else:
             return True
 
     def as_imagecoord(self, header, rot_wrt_axis=1):
-        """
-        Return a new `ShapeList` where the coordinate of the each shape
-        is converted to the image coordinate using the given header
-        information.
+        """New shape list in image coordinates.
+
+        Parameters
+        ----------
+        header : `~astropy.io.fits.Header`
+            FITS header
+        rot_srt_axis : bool
+            TODO: document me
```

Comment:
@leejjoon - It looks like `rot_wrt_axis` is used as a bool flag? OK if I change the default values of 1 / 0 to True / False everywhere? What should be the parameter description (one or two lines)?